### PR TITLE
Had issues installing as the "wheel" package was missing on dietpi :)

### PR DESCRIPTION
**Pull request by rniemand**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_docs/installation/raspberry-pi.markdown [hunk 1]
```diff
@@ -64,6 +64,7 @@ $ source bin/activate
 Once you have activated the virtual environment you will notice the prompt change and then you can install Home Assistant.
 
 ```bash
+(homeassistant) homeassistant@raspberrypi:/srv/homeassistant $ pip install wheel
```
Comment (DubhAd) [inline source/_docs/installation/raspberry-pi.markdown]: For consistency, this should be `pip3`. It's also worth calling out that it's only necessary for DietPi.

Comment (rniemand) [reply source/_docs/installation/raspberry-pi.markdown]: Agreed, as pointed out this is for a different flavor of the OS, but it is a nice to have imho as it opens up a whole lot of new SBC's to run hass. In my case this was an Orange Pi One and hass runs like a beast!

Comment (Landrash) [reply source/_docs/installation/raspberry-pi.markdown]: This page is called raspberry-pi. If it's for other SBC's then we need to create a separate page for those :)

**Pull request closed without merging**